### ISSUE TITLE
Add rdkit package

### ISSUE THREAD

**Issue by vladkisin** [Kaggle/docker-python#531]
It must be useful for many competitions for processing chemoinformatics data, but I've struggeled to install it to a kernel.
https://www.rdkit.org

Would also be pleased to have an insight on how can it be installed to kaggle kernel manually.

**Comment by erdalsivri** [comment]
This was a bit tricky as following the instructions on https://www.rdkit.org/docs/Install.html didn't work on a Kaggle kernel. However, I was able to install it using a single command on an Internet enabled kernel:

```
!conda install -y -c rdkit rdkit
```
This takes around one minute to complete. I confirmed the installation by running the following piece of code:

```python
from rdkit import Chem

m = Chem.MolFromSmiles('Cc1ccccc1')
dir(m)
```

```
['AddConformer',
 'ClearComputedProps',
 'ClearProp',
 'Debug',
...
]
```

Make sure to enable Internet on your kernel before running the installation command.
